umbra-js/src/utils/utils.ts

```diff
@@ -283,9 +283,9 @@ function getResolutionInstance(provider: EthersProvider) {
   const networkName = provider.network.name === 'homestead' ? 'mainnet' : provider.network.name;
   return new Resolution({
     sourceConfig: {
-      cns: {
-        provider: Eip1993Factories.fromEthersProvider(provider),
-        network: networkName as 'mainnet' | 'rinkeby',
+      uns: {
+        api: true,
+        network: networkName === 'mainnet' ? 1 : 4,
```

Comment:
I thought that domains are now minted on polygon, so why are the only two networks used here mainnet and rinkeby?

Reply:
I'm sorry that network line of code is not necessary and i don't think it changes anything about the functionality so I removed it.